drivers/performance_measurement/app/controllers/performance_measurement/warehouse_reports/reports_controller.rb

```diff
@@ -127,11 +128,19 @@ def filter_params
       @pdf_export = PerformanceMeasurement::DocumentExports::ReportExport.new
     end
 
-    def formatted_cell(cell)
+    def formatted_cell(cell, key)
       return view_context.content_tag(:pre, JSON.pretty_generate(cell)) if cell.is_a?(Array) || cell.is_a?(Hash)
       return view_context.yes_no(cell) if cell.in?([true, false])
 
-      cell
+      if key.include?('prior_living_situation')
+        HudUtility.living_situation(cell)
+      elsif key.include?('destination')
+        HudUtility.destination(cell)
+      elsif key.include?('days_')
+        number_with_delimiter(cell)
+      else
+        cell
+      end
```

Comment:
No big deal, but `include` always has me worried it will matching something unexpected... Maybe?

```suggestion
      case key
      when /_prior_living_situation$/
        HudUtility.living_situation(cell)
      when /_destination$/
        HudUtility.destination(cell)
      when /^days_/
        number_with_delimiter(cell)
      else
        cell
      end
```